src/Umbraco.Core/Configuration/Models/SmtpSettings.cs

```diff
@@ -15,6 +15,8 @@ public class SmtpSettings : ValidatableEntryBase
 
         public int Port { get; set; }
 
+        public string SecureSocketOptions { get; set; } = "Auto";
```

Comment:
What are the other possible configuration values here?

Reply:
I meant to flag this one... it should be an enum I agree, but, the enum is defined in `MailKit.Security` which I think breaks the rules on "no external dependencies in `Umbraco.Core`.  So that's why I made it a string on the configuration model.  Thoughts?

Reply:
I found I had to - otherwise smtp4dev that I was testing with locally wouldn't accept the connection and "send" the email.  I've probably got an old copy and maybe there's a bug in it, but the local problem did go away when I changed this setting.

Reply:
The "no dependency" rule falls down hard unless you define only a very limited set of abstractions, which means potentially implementing hundreds of projects as you break down. Personally I think "Clean Architecture" is the equivalent of "in a vacuum" and should not be considered as a practical design goal unless you want hundreds of projects in your solution.

Implementation specific properties really shouldn't exist in the basic config, if you're testing there's no reason why an `ISmptService` or whatever should actually send an email.

Reply:
I've just made the change use a locally defined enum... but will leave this one unresolved for further discussion for now.

Reply:
My alternative was to not expose it as a configuraiton value, and use "Auto" in:
```
client.Connect(_globalSettings.Smtp.Host,
                        _globalSettings.Smtp.Port,
                        MailKit.Security.SecureSocketOptions.Auto)
```

I not sure you will every change that value from auto?

Reply:
Auto is the default, so you don't need to provide it if we do that.  As I say though, in testing locally with smtp4dev, I found "Auto" wouldn't work - gave me an error in trying to set up an SSL connection - and only when I switched it to "None" did it accept the connection.

Reply:
You could always pass the environment as a parameter to the mailkit service and switch to none if on dev?